samples/dect/dect_phy/dect_shell/src/dect/dect_phy_shell.c

```diff
@@ -889,6 +901,28 @@ static int dect_phy_ping_cmd(const struct shell *shell, size_t argc, char **argv
 			params.tx_mcs = atoi(optarg);
 			break;
 		}
+		case DECT_SHELL_PING_TX_LBT_PERIOD: {
+			tmp_value = atoi(optarg);
+			if (tmp_value != 0 && (tmp_value < DECT_PHY_LBT_PERIOD_MIN_SYM ||
+					       tmp_value > DECT_PHY_LBT_PERIOD_MAX_SYM)) {
```

Comment:
This should be enough:
```suggestion
			if (tmp_value < DECT_PHY_LBT_PERIOD_MIN_SYM ||
				tmp_value > DECT_PHY_LBT_PERIOD_MAX_SYM)) {
```